app/uk/gov/hmrc/leakdetection/services/SummaryService.scala

```diff
@@ -120,6 +128,7 @@ class SummaryService @Inject()(ruleService: RuleService,
               case (repoName, ruleLeaksByRepo) =>
                 RepositorySummary(
                   repoName,
+                  isArchived = {if (allArchivedRepos.contains(repoName)) true else false},
```

Comment:
this could be simplified to just `isArchvied = allArchivedRepos.contains(repoName)` since contains returns true/false